R/pkg/inst/tests/test_sparkSQL.R

```diff
@@ -664,10 +664,10 @@ test_that("column binary mathfunctions", {
   expect_equal(collect(select(df, atan2(df$a, df$b)))[2, "ATAN2(a, b)"], atan2(2, 6))
   expect_equal(collect(select(df, atan2(df$a, df$b)))[3, "ATAN2(a, b)"], atan2(3, 7))
   expect_equal(collect(select(df, atan2(df$a, df$b)))[4, "ATAN2(a, b)"], atan2(4, 8))
-  expect_equal(collect(select(df, hypot(df$a, df$b)))[1, "HYPOT(a, b)"], sqrt(1^2 + 5^2))
-  expect_equal(collect(select(df, hypot(df$a, df$b)))[2, "HYPOT(a, b)"], sqrt(2^2 + 6^2))
-  expect_equal(collect(select(df, hypot(df$a, df$b)))[3, "HYPOT(a, b)"], sqrt(3^2 + 7^2))
-  expect_equal(collect(select(df, hypot(df$a, df$b)))[4, "HYPOT(a, b)"], sqrt(4^2 + 8^2))
+  expect_equal(collect(select(df, hypot(df$a, df$b)))[1, "HYPOT(a, b)"], sqrt(1 ^ 2 + 5 ^ 2))
```

Comment:
I agree with you. But how do we deal with the infix warnings about `^`? lintr doesn't support any configurations to exclude infix tokens as follows.
https://github.com/jimhester/lintr/blob/master/R%2Finfix_spaces_linter.R#L1

So as I suggested on JIRA, we will get rid of all the infix space warnings first. And adding a new feature to exclude any infix tokens into lintr would be better. After that, we will modify the infix spaces again. I think it is important to remove all warnings at first.